packages/console/src/pages/Connectors/SocialConnectors.tsx

```diff
@@ -0,0 +1,111 @@
+import { ConnectorDTO, ConnectorType } from '@logto/schemas';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import useSWR from 'swr';
+
+import placeholder from '@/assets/images/social-connectors-placeholder.svg';
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import ImagePlaceholder from '@/components/ImagePlaceholder';
+import ItemPreview from '@/components/ItemPreview';
+import Status from '@/components/Status';
+import TabNav from '@/components/TabNav';
+import TabNavLink from '@/components/TabNav/TabNavLink';
```

Comment:
should we just export `TabNavLink` from `TabNav` then we can `import TabNav, { TabNavLink } from '...'`